packages/core/src/indexing-store/cache.ts

```diff
@@ -465,6 +503,28 @@ export const createIndexingCache = ({
             { method: `${getTableName(table)}.flush()` },
             async () =>
               copy(table, text).catch((error) => {
+                // try to recover the event that caused the error
+                if (
+                  "where" in error &&
+                  (error.where as string).match(/line (\d+)/)
```

Comment:
I think it's possible for `error.where` to be null (or undefined), so we should probably handle that case and throw the original error. Same story for if `where` doesn't match the regex or the matched string cannot be coerced to a number

Maybe
```ts
const where = error.where
if (typeof where !== 'string') throw error 
```

Reply:
I think that is how it works if you read line 533